packaging/installer/kickstart.sh

```diff
@@ -187,13 +183,19 @@ USAGE: kickstart.sh [options]
   --install-version <version>      Specify the version of Netdata to install.
   --claim-token                    Use a specified token for claiming to Netdata Cloud.
   --claim-rooms                    When claiming, add the node to the specified rooms.
-  --claim-only                     If there is an existing install, only try to claim it, not update it.
   --claim-*                        Specify other options for the claiming script.
   --no-cleanup                     Don't do any cleanup steps. This is intended to help with debugging the installer.
-  --uninstall                      Uninstall an existing installation of Netdata.
-  --reinstall-clean                Clean reinstall Netdata.
   --local-build-options            Specify additional options to pass to the installer code when building locally. Only valid if --build-only is also specified.
   --static-install-options         Specify additional options to pass to the static installer code. Only valid if --static-only is also specified.
+
+The following options are mutually exclusive and specifiy special operations other than trying to install Netdata normally or update an existing install:
+
+  --reinstall                      If there is an existing install, reinstall it instead of trying to update it. If there is not an existing install, install netdata normally.
+  --reinstall-even-if-unsafe       If there is an existing install, reinstall it instead of trying to update it, even if doing so is known to potentially break things. If there is not an existing install, install Netdata normally.
+  --reinstall-clean                If there is an existing install, uninstall it before trying to install Netdata. Fails if there is no existing install.
+  --uninstall                      Uninstall an existing installation of Netdata. Fails if there is no existing install.
+  --claim-only                     If there is an existing install, only try to claim it without attempting to update it. If there is no existing install, install and claim Netdata normally.
+  --repositories-only              Only install repository configuration packages instead of doing a full install of Netdata. Automatically sets --native-only.
```

Comment:
Hi Austin.

A couple of thoughts:

1) `--reinstall` & `--reinstall-clean`. Should be the same, besides the fact the latter will do a cleanup as well. But when there is not an existing install, they behave differently. Should they behave the same if no existing install is found?

2) `--claim-only`. I think it should fail if there is not an existing install ?

Also we print the cases for "No existing install" with a couple different ways, i.e. `If there is not an existing install` & `If there is no existing install`. We should use the same I think (The latter seems to me a bit "better") :-)

Reply:
A final question @Ferroin , are you going to modify this PR according these suggestions? :thinking:

Reply:
@thiagoftsm I meant to respond regarding them, but just forgot.

---

@MrZammler 

Things currently behave as you mention because it simplifies the code to have them behave that way. I do intend to fix this eventually, but I would consider such changes to be orthogonal to this PR.

Good catch on the option descriptions though, I agree that those should be consistent, and that the shorter form  is preferable here.